superset/jinja_context.py

```diff
@@ -479,11 +479,11 @@ def __init__(
         self._applied_filters = applied_filters
         self._removed_filters = removed_filters
         self._context: dict[str, Any] = {}
-        self._env = SandboxedEnvironment(undefined=DebugUndefined)
```

Comment:
Given that this PR (and other logic) references `_env` I thought it should be made "public"—in Presto's pseudo public/private naming sense.